internal/controller/http/redfish/v1/errors.go

```diff
@@ -0,0 +1,403 @@
+// Package v1 provides Redfish-compliant error handling utilities.
+package v1
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+
+	redfishapi "github.com/device-management-toolkit/console/redfish/pkg/api"
+)
+
+const (
+	// MessageIDInsufficientPrivilege is the Redfish message ID for insufficient privilege errors.
+	MessageIDInsufficientPrivilege = "Base.1.11.0.InsufficientPrivilege"
```

Comment:
We need to have some kind of lookup map implementation by taking as input the registry name and message code and according generating the Message ID.

For ex: if we want to map an error to the base registry then we need to load the base registry file of 2025.3 release which is Base.1.22.0.json

Look-up for the message ResourceInUse and then use the Severity, Message, Resolution, RegistryPrefix, RegistryVersion
Same should be followed for all the errors in this file.